size_classes.cpp

```diff
@@ -44,7 +44,7 @@ void InitSizeClass()
     for (size_t scIdx = 1; scIdx < MAX_SZ_IDX; ++scIdx) {
         SizeClassData& sc = SizeClasses[scIdx];
         size_t sbSize = sc.sbSize;
-        // 2MB
+        // 16MiB
```

Comment:
Hmm, that explains poor performance in short lived, small programs

Few allocations, few size classes, but massive amounts of memory initialized...